scripts/install-jdk21.sh

```diff
@@ -0,0 +1,13 @@
+# install jdk "manually"
+# depending on os, might want to do something like `apt remove default-jre openjdk-*-jre-*`
+cd /opt
+wget https://download.java.net/java/GA/jdk21.0.1/415e3f918a1f4062a0074a2794853d0d/12/GPL/openjdk-21.0.1_linux-x64_bin.tar.gz
```

Comment:
should write infra policy to cache this as we already do for suse aws jenkins hosts

```
wget -O java.tar.gz http://buildcache.cfengine.com/build-tools/jre-8u151-linux-x64.tar.gz
```

Reply:
this is optional really I would say.